package.json

```diff
@@ -144,7 +144,7 @@
     "postcss-jsx": "^0.36.4",
     "postcss-syntax": "^0.36.2",
     "prettier": "^2.7.1",
-    "puppeteer": "^14.1.1",
+    "puppeteer": "^15.4.1",
```

Comment:
They're moving fast, now at 15.5.0 with latest Chromium

Reply:
Ya for sure

Reply:
Let's update to that one while at it